tests/test_generate_memory_update.py

```
@@ -306,6 +306,65 @@ def test_extra_keys_are_filtered(self):
+        assert "プロジェクト" in result
+
+    def test_cli_output_has_markdown_header(self):
+        """CLI output contains a Claude Memory header."""
+        payload = json.dumps({
+            "repo_name": "test",
+            "repo_type": "single_app",
+            "tech_stack": ["Python"],
+            "key_modules": ["m"],
+            "patterns": [],
+        })
+        result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)